docs/Common-issues.md

```diff
@@ -0,0 +1,14 @@
+## Python Issues OSX
+
+Make sure you are using the native Python version in OSX.  If you use a MacPorts of HomeBrew version, you may run into problems.
```

Comment:
```suggestion
Make sure you are using the native Python version in OSX.  If you use a MacPorts or HomeBrew version, you may run into problems.
```
https://www.macports.org and https://brew.sh are two separate macOS package managers.

I think we should remove these two paragraphs because Apple what’s stated that native Python will be removed from macOS.  I have been using brew Python from the beginning with zero issues and have seen no one complaining about `execvp`.

Reply:
Let’s agree to land this first and then resolve these issues in future PR.